charts/common/Chart.yaml

```diff
@@ -11,3 +11,5 @@ home: https://github.com/k8s-at-home/charts/tree/master/charts/common
 maintainers:
 - name: bjw-s
   email: me@bjw-s.dev
+- name: nicholaswilde
+  email: ncwilde43@gmail.com
```

Comment:
I don't think it's best-practice for every contributor to add their contact details here.

Reply:
@Ornias1993 we're hoping to get @nicholaswilde on-boarded here, so this is fine.

Reply:
> Also: It might be nice naming PR's relating to what they do, instead of just the version change.

Done